src/Raven.Studio/typescript/models/database/tasks/periodicBackup/s3Settings.ts

```diff
@@ -68,18 +82,44 @@ class s3Settings extends amazonSettings {
                 }
             ]
         });
+        
+        this.customServerUrl.extend({
+            required: {
+                onlyIf: () => this.useCustomS3Host()
+            },            
+            validation: [
+                {
+                    validator: (customServerUrl: string) => !this.useCustomS3Host() ||
+                                                            (!this.hasConfigurationScript() && customServerUrl && genUtils.urlRegex.test(customServerUrl)),                    
+                    message: "Expected format: 'http(s)://<yourHostAddress>'"
+                }
+            ]
+        });
 
         this.localConfigValidationGroup = ko.validatedObservable({
             awsAccessKey: this.awsAccessKey,
             awsSecretKey: this.awsSecretKey,
             awsRegionName: this.awsRegionName,
-            bucketName: this.bucketName
+            bucketName: this.bucketName,
+            customServerUrl: this.customServerUrl
         });
     }
+    
+    isRegionRequired() {
+        if (!!this.useCustomS3Host) {
+            return this.enabled() && !this.hasConfigurationScript() && !this.useCustomS3Host();     
+        }
+        else {
+            return this.enabled() && !this.hasConfigurationScript();
```

Comment:
hmm actually we can call parent method here - which check for enabled + check other things here.

Reply:
done